go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -381,10 +382,12 @@ func (m *MasterController) setupHybridLRPolicySharedGw(nodeSubnets []*net.IPNet,
 			Match:    grMatchStr,
 		}
 
-		if err := libovsdbops.CreateOrUpdateLogicalRouterPolicyWithPredicate(m.nbClient, ovntypes.OVNClusterRouter, &grLogicalRouterPolicy, func(item *nbdb.LogicalRouterPolicy) bool {
-			return item.Priority == grLogicalRouterPolicy.Priority &&
-				item.ExternalIDs["name"] == grLogicalRouterPolicy.ExternalIDs["name"]
-		}); err != nil {
+		// TODO(trozet): evaluate if safe to update all fields
```

Comment:
What about this TODO? Making sure you intended to leave it around.

Reply:
fixed when I rebased